.travis_win_deps.bat

```diff
@@ -0,0 +1 @@
+C:\tools\msys64\usr\bin\bash.exe -lc "pacman --needed --noconfirm -S make bison flex mingw-w64-x86_64-llvm mingw-w64-x86_64-clang mingw-w64-x86_64-lld"
```

Comment:
The build failed with`/bin/sh: line 2: cmake: command not found`.

https://github.com/XboxDev/nxdk/wiki/Install-the-Prerequisites must also still be updated for https://github.com/XboxDev/nxdk/pull/144#issuecomment-525822616

Reply:
It's fixed now, and I updated the wiki as well.
I checked that it builds now on Travis, the GitHub check didn't complete yet though (there currently seems to be an issue that prevents the macOS CI VM from starting).